packages/terra-alert/src/Alert.jsx

```diff
@@ -145,7 +132,7 @@ const Alert = ({
     );
   }
   if (onDismiss || action) {
-    bodyClassNameForNarrowParent = cx(['body', 'body-narrow']);
+    bodyClassNameForParent = cx(['body', `alert-parent-${breakpoint}`, 'narrow']);
```

Comment:
probably out of scope, but I wonder if the classNames would be easier to maintain if we inline conditional checks. This way we have assign className values once and eliminate some code duplication.

```jsx
const alertSectionClassName = cx([
  'section', 
  { 'section-custom': type === AlertTypes.CUSTOM },
]);

const bodyClassNameForParent = cx([
  'body', 
  `alert-parent-${breakpoint}`,
  { 'narrow': onDismiss || action },
]);

Reply:
Updated here https://github.com/cerner/terra-core/pull/2747/commits/095c1958ce84835a6d936b112675fa9bddf36951.